docs/slides/content/agile-overview.md

```diff
@@ -75,16 +75,17 @@ continuous delivery of valuable software.
 
 
 
-### Exercise - Pocket Sized Principles
-Working in teams:
+### Exercise - Pocket-sized Principles
+Working in teams
 1. On a piece of paper make room for 12 boxes.
-2. For each box, write 2 to 3 words maximum that effectively capture each of the 12 principles.
+<!--TechEdit: this instruction seems odd, "make room for" on a piece of paper isn't something one can actually do. Give the action directly. "On a piece of paper, draw 12 boxes." -->
```

Comment:
sure, we can go with your suggestion :) 
I'll merge it after your update then ☘️